docs/index.md

```diff
@@ -65,76 +67,50 @@ POST {json file} <url>/insert/
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2016-07-29", "sample_type": "V", "result": "None Detected ug/dL"}, 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2015-08-21", "sample_type": "V", "result": "16.3"}, 
+    {"lab_id": 1234567891011121, "type": "BLL", "date": "2015-02-27", "sample_type": "V", "result": "1"}] 
 }
 ```
 
 ### Response
 
 #### Body
 
-| Field            | Format       | Constraints                                                                     | Notes/Questions                                                                                  |
-|------------------|--------------|---------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|
-| version          | Text         | "0.3.0"                                                                         | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
-| timestamp        | TimeDate     | yyyy-mm-ddThh:mm:ss.sss-hh:mm                                                   | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                               |
-| visit_date       | Date         | yyyy-mm-dd                                                                      | Effective date of score                                                                          |
-| alliance_id      | Text         | AlphaNumeric                                                                    | Return value provided by Feed                                                                    |
-| risk_score       | AlphaNumeric | 9.99                                                                            | Currently expecting numeric, future may be phrased by stating an odds ratio with the risk score. |
-| risk_score_notes | Text         | Additional notes (referral, remediation funds, etc to be returned to provider ) |                                                                                                  |
+| Field            | Format       | Constraints                       | Notes/Questions                                                                                  |
+|------------------|--------------|-----------------------------------|--------------------------------------------------------------------------------------------------|
+| version          | Text         | "0.3.0"                           | Follows Semantic Versioning 2.0.0 http://semver.org/spec/v2.0.0.html                             |
+| timestamp        | TimeDate     | yyyy-mm-dd hh:mm:ss.sss-hh:mm     | Current Timestamp (RFC 3339 complient date).                                                     |
+| patient_id       | Text         | AlphaNumeric                      | Same `patient_id` that was submitted to the API.                                                 |
```

Comment:
After talking with the team today, also need to add `network_id`, `clinic_id`, and `location_id` to the response document since it all needs to be present to make sense.